sysgen/sysgen.go

```diff
@@ -288,10 +288,13 @@ func generateArg(name, typ string, a []string, structs map[string]Struct, unname
 		}
 		fmt.Fprintf(out, "PtrType{%v, Dir: %v, Type: StrConstType{%v, Val: \"%v\"}}", common(), fmtDir("in"), common(), a[0]+"\\x00")
 	case "int8", "int16", "int32", "int64", "intptr":
-		if want := 0; len(a) != want {
-			failf("wrong number of arguments for %v arg %v, want %v, got %v", typ, name, want, len(a))
+		if len(a) == 2 {
```

Comment:
Yes. the form foo int32[0, 5-10, 50, 100-200] is what we want. It is useful for users. I cannot open the link https://play.golang.org/p/CQwd_G6keY   It is forbidden.  Could you help me? Thanks.